doc/examples/realtime_servo/config/panda_simulated_config.yaml

```diff
@@ -0,0 +1,62 @@
+###############################################
+# Modify all parameters related to servoing here
+###############################################
```

Comment:
I'd rather if you got this config file from the servo pkg. One less thing to keep in sync.

https://github.com/ros-planning/moveit2/blob/main/moveit_ros/moveit_servo/config/panda_simulated_config.yaml